app/validators/questionnaire_schema.py

```diff
@@ -117,6 +71,7 @@ def get_context_from_match(match):
     return {"section": section["id"], "block": block_id, "group_id": group["id"]}
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
Can be avoided. Make the new ones added private, as they are not needed externally.

- get_numeric_value
- get_answer_maximum
- get_answer_minimum
- get_numeric_range_values